test/e2e/common_test.go

```diff
@@ -556,8 +556,16 @@ func (p *PodmanTestIntegration) Cleanup() {
 
 	p.StopRemoteService()
 	// Nuke tempdir
-	if err := os.RemoveAll(p.TempDir); err != nil {
-		fmt.Printf("%q\n", err)
+	if isRootless() {
+		// If rootless, os.RemoveAll() is failed due to permission denied
+		cmd := exec.Command(p.PodmanBinary, "unshare", "rm", "-rf", p.TempDir)
```

Comment:
please connect both stdout/err so we can see the error message in the logs, i.e. add
```
cmd.Stdout = os.Stdout
cmd.Stderr = os.Stderr
```